proxy/src/main/java/com/velocitypowered/proxy/command/VelocityCommand.java

```diff
@@ -113,6 +116,11 @@ private Info(ProxyServer server) {
 
     @Override
     public void execute(CommandSource source, String @NonNull [] args) {
+      if (args.length != 0) {
```

Comment:
```suggestion
      if (args.length == 0) {
```

Reply:
No, what he’s suggesting is right, he’s saying that `/velocity info` doesn’t accept any arguments, which is true.